src/components/common/Header.jsx

```diff
@@ -1,53 +1,55 @@
 import React from "react";
+import { emailWhiteIcon, hindfloIcon, phoneWhiteIcon } from "./image";
 
 const Header = () => {
+    let pathName = window.location.pathname;
   return (
     <>
-        <div className="bg-gray-800 border-gray-200 dark:bg-gray-800">
-            <div className="flex items-center text-white px-4 py-4 max-w-screen-xl mx-auto">
-                <div>Dealer Distributor Enquiry</div>
-                <div className="ml-auto">Mobile Number</div>
+        <div className="bg-blue-900 border-gray-200 dark:bg-blue-900">
+            <div className="block sm:flex items-center text-white px-4 py-4 max-w-screen-xl mx-auto">
+                <div className="mb-2 sm:mb-0">Dealer Distributor Enquiry</div>
+                <div className="block sm:flex ml-auto">
+                    <a href="#" className="hover:underline flex mb-2 sm:mb-0 mr-4"><img className="size-6 mr-2" src={emailWhiteIcon} alt="Email"/> hindflo@gmail.com</a>
```

Comment:
Please follow the comments given in footer file

Reply:
done